### ISSUE TITLE
set support: false still showing ads

### ISSUE THREAD

**Issue by testermania** [sdepold/jquery-rss#156]
why still showing ads?

**Comment by testermania** [comment]
6 days nobody reply. this script showing forced ads.

**Comment by sdepold** [comment]
Hey. Sorry for the delay. I’ll have a look at it today!

**Comment by sdepold** [comment]
Can you tell me exactly how you call the library please? That should help with debugging

**Comment by testermania** [comment]
`<script src="//cdnjs.cloudflare.com/ajax/libs/jquery-rss/4.3.0/jquery.rss.min.js"></script>
$(".div").rss("mydomain/rss", {
    limit: 10,
support: false,
    layoutTemplate: '<div class="dl-horizontal">{entries}</div>',
    entryTemplate: '<p><a href="{url}">{title}</a></p>'
}).show()

`

**Comment by sdepold** [comment]
Would you mind sharing the feed URL?

**Comment by testermania** [comment]
I edited the previous message

**Comment by sdepold** [comment]
Turned out it was an issue with my server's hard-drive which caused redis to no longer reset the ads counter 😊Since the experiment didn't work out anyway, I also turned off the entire ads feature altogether. So you shouldn't see any ads anymore.

**Comment by testermania** [comment]
yes, I no longer see ads anymore. I hope you solve the redis problem in a different way.  Thanks!
